src/hooks/useMapBackground.ts

```diff
@@ -0,0 +1,16 @@
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+
+import { useCheckImage } from "hooks/useCheckImage";
+
+export const useMapBackground = (
+  mapBackgroundUrl: string | undefined
+): [string, boolean] => {
+  const { isValid } = useCheckImage(mapBackgroundUrl ?? "");
+
+  return [
+    isValid
+      ? mapBackgroundUrl ?? DEFAULT_MAP_BACKGROUND
+      : DEFAULT_MAP_BACKGROUND,
```

Comment:
Please create a constant for this conditional for better readability.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2002/commits/4f7bca683dd8443d753e222cf319790bf2a047ef